nodepool/elements/nodepool-base/post-install.d/89-jenkins-scripts

```diff
@@ -26,6 +26,12 @@ set -o pipefail
 # /usr/local/jenkins.
 mkdir -p /usr/local/jenkins
 
+# Get the project-config repo
+mkdir -p /opt/git/openstack-infra
+if [ ! -d "/opt/git/openstack-infra/project-config" ]; then
+  (cd /opt/git/openstack-infra && git clone https://git.openstack.org/openstack-infra/project-config)
```

Comment:
you can specify a directory after the git clone command for where to clone it to, I think it’s cleaner to do that rather than cd & clone